source/client/HandView.py

```diff
@@ -2,6 +2,17 @@
 import textwrap
 import client.UIConstants as UIC
 from client.UICardWrapper import UICardWrapper
+import client.ClickableImage as Cli
+import client.Button as Btn
+import operator
+
+# need "operator" for this command:  sorted_x = sorted(x, key=operator.attrgetter('score'))
+# or this one: x.sort(key=operator.attrgetter('score'))
+# Use key to trigger sorting or automatically sort cards when draw cards?
+# Need to think about when to update xy locations of cards and how I want to do it...
+# Want to see what you draw...
+# Two parts to sorting -- one is putting list in order, 2nd part is making xy location recognize list order...
+
```

Comment:
You can almost certainly define an standard comparison operator for cards if you want to sort by card order, if you want to sort by score value then you should use the rules getScore method and sort those numbers with a standard bit. As you don't seem to be using this yet I'm reluctant to leave it in the merge to master (which is supposed to be nice clean code)